src/components/EditDonationModal/EditDonationModal.tsx

```diff
@@ -178,15 +182,18 @@ export const EditDonationModal: React.FC<EditDonationModalProps> = ({
                     name="convertedAmount"
                     value={convertedAmount}
                     label={t('Amount')}
-                    onChange={handleChange}
-                    onBlur={handleBlur}
+                    onChange={(event) => {
+                      setFieldValue('convertedAmount', event.target.value);
+                      setFieldTouched('convertedAmount', true, false);
+                    }}
+                    onBlur={() => {
+                      setFieldTouched('convertedAmount', true);
+                    }}
```

Comment:
You can use handleBlur, and it should work the same.

```suggestion
                    onBlur={(handleBlur}
```